src/components/TransactionSigner.js

```diff
@@ -60,6 +62,17 @@ class TransactionSigner extends React.Component {
         postLink = <a {...postLinkProps}>Submit this transaction to the network</a>;
       }
 
+      let treeView, txDetails;
+      if (result.xdr) { // Only show tree view if xdr is valid and exists
+        let treeView = <TreeView className="TransactionSigner__details__tree" nodes={extrapolateFromXdr(result.xdr, 'TransactionEnvelope')} />
+        let txDetails = <div className="so-back TransactionSigner__details">
+          <div className="so-chunk">
+            <p className="TransactionSigner__details__title">Transaction result details</p>
+            {treeView}
```

Comment:
`treeView` isn't used anywhere else is it?  Perhaps just inline it into `txDetails` declaration and then also remove the higher-scope let statement on line 65.